modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -28,9 +28,20 @@
 import javafx.scene.Node;
 
 /**
- * Base class for defining the visual representation of user interface controls
- * by defining a scene graph of nodes to represent the skin.
- * A user interface control is abstracted behind the {@link Skinnable} interface.
+ * An interface for defining the visual representation of user interface controls.
+ * <p>
+ * A Skin implementation should generally avoid modifying its control outside of
+ * {@link #install()} method.  The life cycle of a Skin implementation
+ * is as follows:
+ * <ul>
+ * <li>instantiation
+ * <li>configuration, such as passing of dependencies and parameters
+ * <li>inside of {@link Control#setSkin(Skin)}:
```

Comment:
Same comment as earlier: replace "inside of Control.setSkin" with "when the skin is set on a Skinnable".